public/locales/en.json

```diff
@@ -373,7 +375,8 @@
         "or": "or",
         "and": "and",
         "avatarOf": "avatar of '{{name}}'",
-        "why": "Why?"
+        "why": "Why?",
+        "successfullyCopied": "Copied!"
```

Comment:
why is this common?

Reply:
it's now used in two very different places

Reply:
Yes, but under "common" are terms that aren't very specific. "successfully copied" is a very specific message.

Reply:
I mean yes.. you use it in two places, then in this case I would create a new top level key instead.

Reply:
What would suggest?

Reply:
TBH: I don't know. Let it stay here until we have a better place.

Reply:
The copyable-field is one of the components using that string. It's located inside our common-components collection. So the translation section "common" is absolutely the right one.